data/transport missions.txt

```diff
@@ -1951,3 +1951,383 @@ ship "Science Drone"
+	
+	on offer
+		conversation
+			`Diana is eager to get going as soon as you land. "This is going to be great! I'm going to do so much good with this! I can't wait to see the look on their faces when I open up my backpack in front of them!" Diana says rapidly as she bounces up and down with the backpack of credit chips on her back.`
+			`	As you gather a few credits to bring with you in case you see something you want to buy, Diana bounces off the ship. Suddenly, her crazed rambling stops. Diana comes running back onto the ship yelling "We need to go! Fly to <destination>! The Free Worlds should protect me!"`
+			choice
+				`	"What is it? What's happening?"`
+				`	(Launch immediately.)`
+					launch
+			
+			`	"Look outside," Diana says with a ghastly look on her face. You check your ship's external cameras, and roaming around your ship is a group of men in Navy uniforms. One of them raises a megaphone to their mouth.`
+			`	"Hand over the girl and her credits and we can all walk away from this peacefully, Captain."`
+			choice
+				`	"Why do you need her?"`
+				`	"Over my dead body!"`
+					goto launch
+				`	"Alright, I'll bring her out."`
+					goto agree
+			
+			`	"In case you haven't noticed, Captain, that girl is in possession of over two billion stolen credits. She's a criminal, and helping her makes you a criminal too. Given your status, Captain <last>, I'd appreciate it if you didn't make me the officer that put <first> <last> behind bars."`
+			choice
+				`	"I'm sorry officer, but I can't help you."`
+					goto launch
+				`	"Okay, I'll hand her over."`
+			
+			label agree
+			`	Diana looks to you in shock. "You wouldn't!" she screeches. "You can't! Fly me to <planet> right now!"`
+			choice
+				`	"I'm not going to become a criminal to fulfill your crazy dream."`
+				`	"Okay, we'll fly to <planet>."`
+					goto launch
+			
+			`	You allow the Navy officer and his troops onto your ship, where they apprehend Diana and her stolen credits. They take her kicking and screaming off of your ship. She yells harsh profanities at you as she is taken away, one among them being "Syndicate lover."`
+			`	"Thank you for not causing any more trouble, Captain," the officer says to you. He then hands you <payment> for "playing nice" and leaves.`
+				decline
+			
+			label launch
+			`	You activate your repulsor engines, sending the Navy troops recoiling away from your ship. As you ascend into space, your computers detect multiple ships targeting you.`
+				launch
+			
+	on accept
+		"reputation: Republic" = -10
+	
+	on decline
+		payment 500000
+	
+	npc evade
+		government "Republic"
+		personality heroic
+		fleet "Large Republic"
+
+
+
+mission "Paradise Fortune 4"
+	landing
+	name `Exchange of Goods`
+	description `Travel to <destination> and hand the backpack of credits and Diana Howl's note over to the Navy.`
+	destination "New Boston"
+	to offer
+		has "Paradise Fortune 3: done"
+	
+	on offer
+		"reputation: Republic" = 10
```

Comment:
I wouldn't set this in the offer, but rather an `on accept`

This way if a player starts this mission set with a large negative Republic rep, they will keep this rep if they steal the credits, rather than getting bumped positive.